app/state_machines/configurable_state_machine/machine.rb

```diff
@@ -8,6 +8,10 @@ def initialize(config:, kase:)
       @events = nil
     end
 
+    # Ideal there shouldn't be any other definitions of states other than state machine yaml file. but 
+    # tried to remove it with a method in manager and ended up with an error below 
+    # >> Circular dependency detected while autoloading constant in one of policy ruby code
+    # Couldn't think of easy way to fix it, so just added missing states here
```

Comment:
Interesting - I think this would be good to investigate - perhaps we should pair on it 🤔